pya/amfcc.py

```diff
@@ -3,15 +3,9 @@
 from .helper import next_pow2, signal_to_frame, round_half_up, magspec
 from .helper import mel2hz, hz2mel, is_pow2
 from .helper import basicplot
-import logging
 from scipy.signal import get_window
 from scipy.fftpack import dct
 import pya.asig
-import matplotlib.pyplot as plt
-from mpl_toolkits.axes_grid1 import make_axes_locatable
-
-_LOGGER = logging.getLogger(__name__)
-_LOGGER.addHandler(logging.NullHandler())
 
```

Comment:
see my comment why i think having loggers with file scopes is better than a global module logger

Reply:
Make sense, will drop